etl_tests/resources/test_open_ai_embedding_model_resource.py

```diff
@@ -0,0 +1,31 @@
+from langchain.embeddings import CacheBackedEmbeddings
+from langchain_community.vectorstores import FAISS
+from pytest_mock import MockFixture
+
+from etl.models import wikipedia
+
+
+def test_create_embedding_store(
+    session_mocker: MockFixture,
+    tuple_of_article_with_summary: tuple[wikipedia.Article, ...],
+    openai_embedding_model_resource,
+) -> None:
+    """Test that OpenAiEmbeddingPipeline.create_embedding_store calls methods that are needed to create an embedding store."""
```

Comment:
The extent of mocking in this test function suggests design issues in the classes, noted above.

Reply:
Yes, I have redesigned the LLM classes.